core/domain/user_id_migration.py

```diff
@@ -427,8 +424,9 @@ def entity_classes_to_map_over(cls):
     def map(user_model):
         """Implements the map function for this job."""
         gae_id = user_model.gae_id
-        if (len(user_model.id) != user_models.USER_ID_LENGTH or
-                not all(c.islower() for c in user_model.id)):
+        if (not user_model.id.startswith('uid_') or
```

Comment:
I suggest you write a standalone function that checks whether a user ID is in the correct "new" format, and write unit tests for that function as well. Then call that function here.

Reply:
Done.